components/TheySaid/index.vue

```diff
@@ -0,0 +1,317 @@
+                    'they-said__info-arrow-right--disabled': currIndex === (contents.length - 1),
+                  }"
+                  size="20px"
+                  name="chevron-right"
+                />
+              </div>
+            </div>
+          </div>
+          <div class="they-said__slides">
+            <div class="they-said__slides-images">
+              <transition name="slide-fade" mode="out-in">
+                <div :key="content.image">
+                  <img class="they-said__slides-image" :src="content.image" :alt="content.name">
+                </div>
+              </transition>
+            </div>
+            <div
+              :class="{
+                'they-said__slides-btn-left': true,
+                'they-said__slides-btn-left--disabled': currIndex === 0
+              }"
+              @click="prevSlide"
+            >
+              <jds-icon
+                :class="{
+                  'they-said__slides-arrow-left': true,
+                  'they-said__slides-arrow-left--disabled': currIndex === 0,
+                }"
+                size="20px"
+                name="chevron-left"
+              />
+            </div>
+            <div
+              :class="{
+                'they-said__slides-btn-right': true,
+                'they-said__slides-btn-right--disabled': currIndex === (contents.length - 1)
+              }"
+              @click="nextSlide"
+            >
+              <jds-icon
+                :class="{
+                  'they-said__slides-arrow-right': true,
+                  'they-said__slides-arrow-right--disabled': currIndex === (contents.length - 1),
+                }"
+                size="20px"
+                name="chevron-right"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'TheySaid',
+  data () {
+    return {
+      currIndex: 0,
+      content: {},
+      contents: [
+        {
+          name: 'Indah Dwianti',
+          role: 'Kepala Seksi Aplikasi dan Data Digital Jabar Digital Service',
+          feedback: 'Lewat Program Desa Digital, diharapkan kedepannya bisa menambah jumlah desa mandiri di Jawa Barat.',
+          image: require('~/assets/images/about/indah.png')
+        },
+        {
+          name: 'Rizi Advista Permana',
+          role: 'Project Officer Jabar Digital Service',
+          feedback: 'Semoga kesejahteraan masyarakat dapat tercapai dan Desa Digital bisa menjadi acuan bagi instansi lain yang ingin menerapkan program serupa.',
+          image: require('~/assets/images/about/rizi.png')
+        },
+        {
+          name: 'Cynditia Fitriani Rahardjo',
+          role: 'Project Officer Jabar Digital Service',
+          feedback: 'Desa Digital adalah desa yang bisa memanfaatkan teknologi untuk meningkatkan produktivitas dan kesejahtteraan masyarakatnya secara mandiri.',
```

Comment:
type `kesejahtteraan` 😅

Reply:
thanks for correction

Reply:
i fixed on this https://github.com/jabardigitalservice/dedi-landing-page/pull/66/commits/050079084f95edee619552d28c34b1d6f964af8a